plugins/cortex_v2/plugin.spec.yaml

```diff
@@ -8,7 +8,7 @@ support: community
 status: []
```

Comment:
Only suggestion would be to add supported versions to plugin.spec.yaml and help.md.